File: 백준/Silver/1105. 팔/팔.py
a, b = map(str, input().rstrip().split())
if len(a) < len(b):
    print(0)
else:
    if a == b:
        print(a.count('8'))
    else:
        cnt = 0
        for i in range(len(a)):
            if a[i] == b[i]:
                if a[i] == '8':
                    cnt +=1
                continue
            else:
                break
        print(cnt)

File: 백준/Silver/1105. 팔/팔.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split(' ');

const a = input[0];
const b = input[1];

if (a.length < b.length) {
    console.log(0);
} else {
    if (a === b) {
        console.log(a.split('8').length - 1);
    } else {
        let cnt = 0;
        for (let i = 0; i < a.length; i++) {
            if (a[i] === b[i]) {
                if (a[i] === '8') {
                    cnt++;
                }
                continue;
            } else {
                break;
            }
        }
        console.log(cnt);
    }
}
